### PR TITLE
Add guide on exporting zonefile from Windows Server

### PR DESCRIPTION
**Pull request by prees1**
This step is critical for customers using Windows DNS, in order to use `tidal discover`. Documenting it here so we can easily reference it during the process.

### CODE REVIEW

File: pages/tidal tools/zone-export.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Exporting a Zonefile from Windows DNS
+keywords: zonefile, discoverym bind, zone files, dns
```
Comment (kekekoko) [inline pages/tidal tools/zone-export.md:3]:
```suggestion
keywords: zonefile, discovery bind, zone files, dns
```

File: pages/tidal tools/zone-export.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- `zonename`	Specifies the name of the zone.
+- `zoneexportfile`	Specifies the name of the file to create.
+
+There is also the [offical documentation](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/dnscmd#dnscmd-zoneexport-command) from Microsoft.
+
+This command will generate a file that is named, what you provided for `zoneexportfile` and it will be located in the `%systemroot%/System32/Dns` directory by defaullt.
+
+# DNS Discovery
+
+Once you have a zonefile generated. You can continue with your discovery by following the [DNS Discovery Guide](/discover.html).
```
Comment (kekekoko) [inline pages/tidal tools/zone-export.md:28]:
```suggestion
Once you have a zonefile generated, you can continue with your discovery by following the [DNS Discovery Guide](/discover.html).
```

File: pages/tidal tools/zone-export.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+```
+
+The arguments for the command are:
+- `servername`	Specifies the DNS server to manage, represented by IP address, FQDN, or host name. If this parameter is omitted, the local server is used.
+- `zonename`	Specifies the name of the zone.
+- `zoneexportfile`	Specifies the name of the file to create.
+
+There is also the [offical documentation](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/dnscmd#dnscmd-zoneexport-command) from Microsoft.
+
+This command will generate a file that is named, what you provided for `zoneexportfile` and it will be located in the `%systemroot%/System32/Dns` directory by defaullt.
```
Comment (kekekoko) [inline pages/tidal tools/zone-export.md:24]:
```suggestion
This command will generate a file that is named what you provided for `zoneexportfile` and it will be located in the `%systemroot%/System32/Dns` directory by defaullt.
```
Not sure about this one but the comma sounds wrong

**Review by kekekoko — COMMENTED** [review]
Looks good, I just added some very minor suggestions but might be wrong because I'm not a native speaker.

File: pages/tidal tools/zone-export.md (hunk 3 above)
Comment (prees1) [reply pages/tidal tools/zone-export.md:24]: hm yea. the suggested comma there breaks at the wrong spot. Why did you think it needed one? Was it hard to follow or too long?

Comment (kekekoko) [reply pages/tidal tools/zone-export.md:24]: The comma was there and I suggested to remove it. So yea I'm all for removing it 👍

Comment (prees1) [reply pages/tidal tools/zone-export.md:24]: oh yikes. 💯

**Review by kekekoko — APPROVED** [review]
Looks great! 🚀
